.github/workflows/ci.yaml

```
@@ -964,6 +964,109 @@ jobs:
         run: |
           ./script/check_dirty
 
+  pytest-postgres:
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres:15.0
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_PASSWORD: password
+        options: --health-cmd="pg_isready -hlocalhost -upostgres" --health-interval=5s --health-timeout=2s --health-retries=3
+    if: |
+      (github.event_name != 'push' || github.event.repository.full_name == 'home-assistant/core')
+      && github.event.inputs.lint-only != 'true'
+      && (needs.info.outputs.test_full_suite == 'true' || needs.info.outputs.tests_glob)
```

We only need this when the recorder tests run (which is always a full suite).

```suggestion
      && needs.info.outputs.test_full_suite == 'true'
```

---

👍 
The same is needed also for the MariaDB job.